src/Backend/Modules/Tags/Entity/Tag.php

```diff
@@ -0,0 +1,208 @@
+ */
+class Tag
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", length=11)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $language;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $tag;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", length=11)
+     */
+    private $number;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $url;
+
+    /**
+     * @ORM\OneToMany(targetEntity="Backend\Modules\Tags\Entity\TagConnection", mappedBy="tag", cascade={"persist", "remove"})
+     */
+    private $connections;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->connections = new \Doctrine\Common\Collections\ArrayCollection();
+    }
+
+    /**
+     * Gets id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Gets the value of language.
+     *
+     * @return string
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * Sets the value of language.
+     *
+     * @param string $language the language
+     * @return self
+     */
+    public function setLanguage($language)
+    {
+        $this->language = $language;
+
+        return $this;
+    }
+
+    /**
+     * Gets the value of tag.
+     *
+     * @return string
+     */
+    public function getTag()
+    {
+        return $this->tag;
+    }
+
+    /**
+     * Sets the value of tag.
+     *
+     * @param string $tag the tag
+     * @return self
+     */
+    public function setTag($tag)
+    {
+        $this->tag = $tag;
+
+        return $this;
+    }
+
+    /**
+     * Gets the value of number.
+     *
+     * @return integer
+     */
+    public function getNumber()
+    {
+        return $this->number;
+    }
+
+    /**
+     * Sets the value of number.
+     *
+     * @param string $number the number
+     * @return self
+     */
+    public function setNumber($number)
+    {
+        $this->number = $number;
+
+        return $this;
+    }
+
+    /**
+     * Gets the value of url.
+     *
+     * @return integer
+     */
+    public function getUrl()
+    {
+        return $this->url;
+    }
+
+    /**
+     * Sets the value of url.
+     *
+     * @param string $number the url
+     * @return self
+     */
+    public function setUrl($url)
+    {
+        $this->url = $url;
+
+        return $this;
+    }
+
+    /**
+     * Add connections
+     *
+     * @param \Backend\Modules\Tags\Entity\TagConnection $connections
+     * @return Category
+     */
+    public function addConnection(\Backend\Modules\Tags\Entity\TagConnection $connections)
+    {
+        $this->connections[] = $connections;
```

Comment:
This method should contain `$connection->setTag($this);` to make sure the coupling is done in both ways.

I'd also name the variable send to this method "$connection" since you add only one connection at once.